### PR TITLE
Add transformer-deploy project

### PR DESCRIPTION
**Pull request by pommedeterresautee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,6 +184,7 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Redis-AI](https://github.com/RedisAI/RedisAI) ![](https://img.shields.io/github/stars/RedisAI/RedisAI.svg?style=social) - A Redis module for serving tensors and executing deep learning models. Expect changes in the API and internals.
 * [Seldon Core](https://github.com/SeldonIO/seldon-core) ![](https://img.shields.io/github/stars/SeldonIO/seldon-core.svg?style=social) - Open source platform for deploying and monitoring machine learning models in kubernetes - [(Video)](https://www.youtube.com/watch?v=pDlapGtecbY)
 * [Tempo](https://github.com/SeldonIO/tempo) ![](https://img.shields.io/github/stars/SeldonIO/tempo.svg?style=social) - Open source SDK that provides a unified interface to multiple MLOps projects that enable data scientists to deploy and productionise machine learning systems.
+* [Transformer-deploy](https://github.com/ELS-RD/transformer-deploy/) ![](https://img.shields.io/github/stars/ELS-RD/transformer-deploy.svg?style=social) - Transformer-deploy is an efficient, scalable and enterprise-grade CPU/GPU inference server for Hugging Face transformer models.
```
Comment (axsaucedo) [inline README.md:187]: The list is alphabetic - this would just need to be sorted accordingly

**Review by axsaucedo — CHANGES_REQUESTED** [review]
Thank you for the contribution - looks good, only thing needs to be alphabetical order

**Comment by pommedeterresautee** [comment]
Tks for your review. It's fixed.

**Comment by axsaucedo** [comment]
Nice one - thanks

**Pull request merged**